sentry-ruby/lib/sentry/transport.rb

```diff
@@ -173,6 +168,19 @@ def fetch_pending_client_report
 
       [item_header, item_payload]
     end
+
+    def process_rate_limits(envelope)
```

Comment:
I think `reject_rate_limited_items` gives a better intention here. Or it's possible to extend this method in the future?